### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by tituspijean**
- Add CPE in manifest
- Relative path for `--keep`
- Use sandbox domain
- Upgrade to v5.1.0

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by tituspijean** [comment]
!testme

**Comment by Ddataa** [comment]
By the power of systemd, I invoke The Great App CI to test this Pull Request!

**Review by ericgaspar — APPROVED** [review]

**Comment by ericgaspar** [comment]
merging?

**Comment by Ddataa** [comment]
I think more testing will be ok, i am not sure if the jq part to rewrite conf.persistent is foolproof...

**Comment by ericgaspar** [comment]
Are we readyyy to merge?

**Comment by ericgaspar** [comment]
!testme

**Comment by Ddataa** [comment]
the checksum of release is wrong, it shall be this `945abe5bae0da25a4e2ef8e02730aaa5bb5e5a0b8bfd7a23a09ec38422d7c47f`

I don't have write right on this repo. Shall I need to do another fork/PR or someone can just update it...

**Comment by Ddataa** [comment]
I have tested the install this morning it was correct and now I am testing upgrading.

Anyone can do more tests please it will be great.

**Comment by Ddataa** [comment]
Removing / upgrading is working too.

I haven't tested the change or url and I will be betting it doesnt work as is...

**Comment by rodinux** [comment]
> I have tested the install this morning it was correct and now I am testing upgrading.
> 
> Anyone can do more tests please it will be great.

Hello, it will be nice if it works ! I gonna try on a local virtual yunohost

**Comment by rodinux** [comment]
It seems working, I can't really verify on local virtualbox environment, because I have not possibility to configure the lest'encrypt... 
Can you please also add a `--keep="/customize" for the script upgrade ? It is the directory to customize the app...

**Comment by Ddataa** [comment]
@rodinux thanks for testing and your feedback !!
I saw you already add the customize folder in your PR https://github.com/YunoHost-Apps/cryptpad_ynh/pull/160

we might be ready to merge and ship then ?!

**Review by ericgaspar — APPROVED** [review]

**Comment by ericgaspar** [comment]
@Ddataa shall we merge to master?

File: manifest.json [hunk 1]
```diff
@@ -22,7 +23,7 @@
         "url": "https://frju365.yunohost.support"
     },
     "requirements": {
-        "yunohost": ">= 4.3.0"
+        "yunohost": ">= 11.0.9"
```
Comment (ericgaspar) [inline manifest.json:26]:
```suggestion
        "yunohost": ">= 11.1.6"
```

**Review by ericgaspar — COMMENTED** [review]
I wonder if requirements to the last version is not necessary...

**Comment by Ddataa** [comment]
> @Ddataa shall we merge to master?

yes we shall

File: manifest.json (hunk 1 above)
Comment (Ddataa) [reply manifest.json:26]: yes it is necessary since the bug was solved that was stopping us from having a sandbox domain setup correctly

Comment (Ddataa) [reply manifest.json:26]: can you make the change ?

File: conf/config.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ module.exports = {
  *
  *  CUSTOMIZE AND UNCOMMENT THIS FOR PRODUCTION INSTALLATIONS.
  */
-    // httpSafeOrigin: "https://some-other-domain.xyz",
+    httpSafeOrigin: "https://sandbox-__DOMAIN__",
```
Comment (Ddataa) [inline conf/config.js:75]: this is not correct, it shall be __SANDBOXDOMAIN__ like in the nginx conf

File: check_process [hunk 3]
```diff
@@ -23,3 +25,6 @@ Notification=none
 ;;; Upgrade options
 	; commit=2a54cd03f90c93b07150a64644ffc7f208110a18
 		name=update to 4.10.0
+;;; Upgrade options
```
Comment (Ddataa) [inline check_process:28]: I dunno about those...

**Review by Ddataa — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**